File: Programming Basics with Python/06 Nested loops/More exercises/05_Challenge_the_Wedding.py
men = int(input())
women = int(input())
max_places = int(input())
counter = 0
no_more_tables = False
for man in range(1, men + 1):
    for woman in range(1, women + 1):
        counter += 1
        print(f"({man} <-> {woman})", end=" ")
        if counter == max_places:
            no_more_tables = True
            break
    if no_more_tables:
        break

File: JS Basics/06 Nested Loops - More Exercises/05. Challenge the Wedding.js
function challenge(input) {
    let men = Number(input[0]);
    let women = Number(input[1]);
    let places = Number(input[2]);
    let print = "";
    let counter = 0;
    let noPlaces = false;
    for (let man = 1; man <= men; man++) {
        for (let woman = 1; woman <= women; woman++) {
            counter++;
            print += `(${man} <-> ${woman}) `;
            if (counter == places) {
                noPlaces = true;
                break;
            }
        }

        if (noPlaces == true) {
            break;
        }
    }
    console.log(print);
}
challenge([5, 8, 40])
